### ISSUE TITLE
Disable or configure nesting rule does not work in Xcode

### ISSUE THREAD

**Issue by MadGeorge** [realm/SwiftLint#3218]
### New Issue Checklist

- [x] Updated SwiftLint to the latest version
- [x] I searched for [existing GitHub issues](https://github.com/realm/SwiftLint/issues)

### Describe the bug

Attempt to disable or change configuration for rule `nesting` has no effect in Xcode, but works in terminal.

I tried disable rule
```yml
disabled_rules:
  - nesting
```

And configure
```yml
nesting:
  type_level: 5
  statement_level: 5
```

With same effect. 

```bash
$ swiftlint version
$ 0.39.2

$ swiftlint rules | grep nest
$ | nesting     | no     | no      | no     | metrics     | no      | (type_leve... |

$ swiftlint lint
$ outputs no violations related to the nesting
```

When rule disabled directly in specific file via comment `// swiftlint:disable nesting` at the beginning of the source file it remove Xcode warnings.

In other cases i have clear warnings 
![CleanShot 2020-05-27 at 16 47 13](https://user-images.githubusercontent.com/25031713/83030508-45ef1600-a03c-11ea-8b28-4ca97c498999.png)

### Environment

* SwiftLint version 0.39.2
* CocoaPods 1.9.2
* Paste your configuration file:

```yml
excluded:
  - SomeTests
  - SomeExtensionsTests
  - SomeHelpersTests

disabled_rules:
  - force_cast
  - force_try
  - nesting # has no effect in xcode

large_tuple:
  warning: 4
  error: 5
```

* [Nested configurations](https://github.com/realm/SwiftLint#nested-configurations) does not used
* Xcode 11.5 11E608c

Issue persist after restarting Xcode, cleaning build folder, removing DerivedData folder and restarting mac.

**Comment by marcelofabri** [comment]
Can you provide a sample project that shows the issue?

**Comment by MadGeorge** [comment]
@marcelofabri it looks like i can't. On new clean project with minimal example it works as expected. I can only reproduce this on my working project (over 2k files). I will investigate it deeply. May be closed for now. I will resubmit new bug with example project if any. Thanks.
